src/components/announcements-list/AnnouncementsList.tsx

```diff
@@ -0,0 +1,105 @@
+import { Row } from 'antd';
```

Comment:
The connection to Redux in this component should be moved to the Announcements container, and this component should simply take a list of announcements. Components should be simple, taking props and displaying data, while containers should be used for data operations.